packages/contracts/src/governance/MainVotingPlugin.sol

```diff
@@ -10,10 +10,18 @@ import {MajorityVotingBase} from "./base/MajorityVotingBase.sol";
 import {IMembers} from "./base/IMembers.sol";
 import {IEditors} from "./base/IEditors.sol";
 import {Addresslist} from "./base/Addresslist.sol";
+import {SpacePlugin} from "../space/SpacePlugin.sol";
 
 // The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
 bytes4 constant MAIN_SPACE_VOTING_INTERFACE_ID = MainVotingPlugin.initialize.selector ^
     MainVotingPlugin.createProposal.selector ^
+    MainVotingPlugin.proposeEdits.selector ^
+    MainVotingPlugin.proposeAcceptSubspace.selector ^
+    MainVotingPlugin.proposeRemoveSubspace.selector ^
+    MainVotingPlugin.addEditor.selector ^
+    MainVotingPlugin.removeEditor.selector ^
+    MainVotingPlugin.addMember.selector ^
+    MainVotingPlugin.removeMember.selector ^
```

Comment:
Is there any reason we don't have `propose` wrappers for adding/removing editors or members?

Reply:
I was about to remove these interface ID's. Are you making use of them?

Reply:
We are for editors

Reply:
@baiirun The editor proposal wrappers have been added. What I meant is about the interface ID, are you using it anyhow?

Reply:
Nope, we're not using the interface IDs